### ISSUE TITLE
F-Droid inclusion

### ISSUE THREAD

**Issue by momobobe** [Jasper-1024/DeepSleep#5]
Hello,

Since the app is open source licensed, would you agree to have it included to [F-Droid](https://f-droid.org/)? It's a strong community (actually almost the unique) focusing on distributing open-source Android apps.

You can find documentation [here](https://f-droid.org/docs/).

If you're interested, feel free to contact @ linsui or @ Poussinou for further details.

Thanks for your attention and it wouldn't take too much time since their team would help you.

PS: Apps on F-Droid are signed by F-Droid by default, but you can ask for the case of reproducible build, then they just verify if your builds are reproducible and simply distribute the builds signed by you, in that way there would be no problem of distributing packages with different signatures.

**Comment by Jasper-1024** [comment]
Yes, I am planning to do this. after rencently what happen about M66B, i decide just use F-Droid.
so not only deepsleep but also nowakelock will add to F-Droid.

However, I have to say that the deepsleep implementation is buggy and the current fix causes incompatibility with previous versions, i am working on it.

**Comment by Jasper-1024** [comment]
I have to say that this process is very complicated and cannot be completed in a short time.😒

**Comment by momobobe** [comment]
Just found this issue still open and I'd like to help it done. However, being not a developer, I can only provide some info that I know to my best. The publishing process has been sorta simplified so far, and now you can try to use your own signature instead of F-Droid's, which means if your build is reproducible, they'll simply pick your released apk. I don't know too much about the details, but if you're still motivated, feel free to contact their contributors for help, among whom one friendly and very nice guy is [linsui](https://github.com/linsui).

**Comment by Jasper-1024** [comment]
> Just found this issue still open and I'd like to help it done. However, being not a developer, I can only provide some info that I know to my best. The publishing process has been sorta simplified so far, and now you can try to use your own signature instead of F-Droid's, which means if your build is reproducible, they'll simply pick your released apk. I don't know too much about the details, but if you're still motivated, feel free to contact their contributors for help, among whom one friendly and very nice guy is [linsui](https://github.com/linsui).

Thank you my friend, this change will be completed in May.

I will be graduating in June 23 and have been busy looking for a job for the first half of the year. . So blog and project updates are on hold.

The employment situation in China is very bad this year, and I didn't find a suitable job until recently.

**Comment by Jasper-1024** [comment]
will done this mouth
